Tools/peg_generator/pegen/build.py

```diff
@@ -89,12 +90,12 @@ def compile_c_extension(
     static library of the common parser sources (this is useful in case you are
     creating multiple extensions).
     """
-    import setuptools.logging
+    import setuptools.logging  # type: ignore[import]
 
     from setuptools import Extension, Distribution
-    from setuptools._distutils.dep_util import newer_group
-    from setuptools._distutils.ccompiler import new_compiler
-    from setuptools._distutils.sysconfig import customize_compiler
+    from setuptools._distutils.dep_util import newer_group  # type: ignore[import]
+    from setuptools._distutils.ccompiler import new_compiler  # type: ignore[import]
+    from setuptools._distutils.sysconfig import customize_compiler  # type: ignore[import]
```

Comment:
If you're planning to start installing `types-setuptools` in CI as a followup PR anyway, why not just add this to the mypy config for now?

```cfg
[mypy-setuptools.*]
ignore_missing_imports = True
```

It would reduce the amount of churn in this file

Reply:
It generates a lot of warnings.

Reply:
Even so, telling mypy to ignore all missing imports from setuptools feels cleaner than `type: ignor`ing each import individually: there's much less churn required